tests/Features/RedirectTest.php

```diff
@@ -34,3 +34,20 @@
     expect(tenant_route('foo.localhost', 'foo', ['a' => 'as', 'b' => 'df']))->toBe('http://foo.localhost/abcdef/as/df');
     expect(tenant_route('foo.localhost', 'foo', []))->toBe('http://foo.localhost/abcdef');
 });
+
+test('redirect from central to tenant works fine', function () {
```

Comment:
Does this test fail when the feature isn't in the config? It might be a good idea to assert this using a dataset.

Regarding the name:
1. The name should be more descriptive. We don't really care about central/tenant here, we care that it can be used before `tenancy()` is instantiated
2. So I guess we should also check that `Tenancy` is not resolved in the container (not sure what the default lifecycle is for tests here)
3. I wouldn't use `fine` in a test name
4. About point 1): Since this is in `RedirectTest`, it might make sense to use language that's more specific to the feature (`redirect from central to tenant`), rather than what I suggested in 1). But we do want a test that verifies that *the bug itself* is resolved.

So maybe a good solution could be to use a name that's specific to this feature, and add a comment explaining that this fixes a bug?
```suggestion
// Check that `domain()` can be called on a redirect before tenancy is used (regression test for #949)
test('redirect from central to tenant works', function () {
```

Reply:
> Does this test fail when the feature isn't in the config?

Yes, it does. 
I see TenancyServiceProvider runs before we add this feature in the config array _in_ tests. So feature never bootstraps. That's why I have to keep the feature in the config already. 

> It might be a good idea to assert this using a dataset.

Do you mean testing both sides? central to the tenant and vice versa?

Reply:
> Does this test **fail when the feature isn't in the config**? It might be a good idea to assert `this` using a dataset.

`this` being the highlighted part

Reply:
I believe this review is resolved now.